tdvt/tdvt/exprtests/regexcalcs/expected.setup.string.regexp.extract.full.txt

```diff
@@ -0,0 +1,212 @@
+<results>
```

Comment:
I think the corresponding test file (`tdvt/tdvt/exprtests/regexcalcs/setup.string.regexp.extract.full.txt`) has been removed because it's redundant? If so can remove this file. If not, ignore me 😄